### PR TITLE
ViewTester fixes

### PR DESCRIPTION
**Pull request by DarkSide666**
* fix interactive / loader
* fix interactive / modal

### CODE REVIEW

**Comment by mvorisek** [comment]
~I do not care about Docker, I mean ok with me, but~ do we really need to include tests for 1, 2 demos? Seems like bad practice too me so I advise to not do so.

**Comment by DarkSide666** [comment]
Yeah I was thinking about moving that class to demos/ViewTester, because it wasn't used anywhere else. But then again - maybe some other repos use it somewhere...
OK, who cares - you're right.

**Comment by mvorisek** [comment]
I mean to not include any tests from demos. Tests should be possible to delete without loosing any functionality. These are unwritten rules for good design. (I see my comment was not very clear, I updated it now)

**Comment by DarkSide666** [comment]
Check out now.

File: demos/ViewTester.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
```
Comment (mvorisek) [inline demos/ViewTester.php]: Move probably to `demos/_includes/`

**Comment by mvorisek** [comment]
Ok to merge

**Pull request merged**